org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/panel/BroadcastMessageWindow.java

```diff
@@ -119,6 +123,12 @@ private void init() {
 		htmlDiv.setStyle("display: table-cell; vertical-align: middle; text-align: center;");
 		Div divAlign = new Div();
 		divAlign.setStyle("color:white; position: absolute; width: 370px; height: 120px; display: table;");
+
+		Env.setContext(Env.getCtx(), MBroadcastMessage.CLIENTINFO_BROADCAST_COMPONENT_ID, pnlHead.getUuid());
+		textMsgContent.setContent(mbMessages.get(0).get_Translation(MBroadcastMessage.COLUMNNAME_BroadcastMessage));
+		pnlHead.addEventListener(ZoomEvent.EVENT_NAME, this);
+		htmlDiv.setFocus(true);
```

Comment:
this added block should replace "textMsgContent.setContent(mbMessages.get(0).get_Translation(MBroadcastMessage.COLUMNNAME_BroadcastMessage));" at line 117.

Reply:
done